### ISSUE TITLE
pre_train_model

### ISSUE THREAD

**Issue by Risemxd** [leftthomas/SuperResolution#29]
@Risemxd 
Duplicate of #25

I ran train.py, but only printed Train Loss .. and Val Loss .. , But the statement:" torch. Save (model.state_dict(), 'epochs/epoch_%d_% d.epot '% (UPSCALE_FACTOR, state['epoch']) " did not run.
